src/__tests__/byDisplayValue.test.js

```diff
@@ -100,3 +100,34 @@ test('findBy queries work asynchronously', async () => {
   await expect(findByDisplayValue('Display Value')).resolves.toBeTruthy();
   await expect(findAllByDisplayValue('Display Value')).resolves.toHaveLength(1);
 }, 20000);
+
+test('all queries should respect accessibility', async () => {
+  const Comp = () => (
+    <View>
+      <View accessibilityElementsHidden>
+        <TextInput value="test_01" />
+      </View>
+      <TextInput value="test_02" />
```

Comment:
Pls add some more `expects` here that query test_02 so that we know that `accessibilityElementsHidden` did not affect sibling elements.